src/app/ReviewService/ReviewConditionsBuilder.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System;
+using System.Collections.Generic;
+using ReviewService.Abstractions;
+
+namespace ReviewService;
+
+[System.Diagnostics.CodeAnalysis.SuppressMessage("StyleCop.CSharp.MaintainabilityRules", "SA1402:File may only contain a single type", Justification = "Using same class name, one is generic and the other is not.")]
+
+/// <summary>
+/// Implementation of <see cref="IReviewConditionsBuilder{TReviewSettings}"/>.
+/// </summary>
+/// <typeparam name="TReviewSettings">The type of the object that we use for tracking.</typeparam>
+public sealed class ReviewConditionsBuilder
```

Comment:
Shouldn't this be a static class?
(If it does, make sure you update the summary.)